invenio_curations/ext.py

```diff
@@ -81,3 +83,30 @@ def init_resources(self, app):
             service=self.curations_service,
             config=CurationsResourceConfig.build(app),
         )
+
+    @property
+    def allow_publishing_edits(self):
+        """Get the configured value of ``CURATIONS_ALLOW_PUBLISHING_EDITS``."""
+        return current_app.config.get("CURATIONS_ALLOW_PUBLISHING_EDITS", False)
+
+    @property
+    def moderator_permissions_via_grants(self):
+        """Get the configured value of ``CURATIONS_PERMISSIONS_VIA_GRANTS``."""
+        return current_app.config.get("CURATIONS_PERMISSIONS_VIA_GRANTS", True)
+
+    @property
+    def moderation_role_name(self):
+        """Get the configured name of the ``CURATIONS_MODERATION_ROLE``."""
+        role = current_app.config["CURATIONS_MODERATION_ROLE"]
+
+        if isinstance(role, Role):
+            # As of InvenioRDM v12, the role's name and ID should be the same anyway
+            return role.name or role.id
+
+        # If it's not a role, we assume it's a string
+        return role
+
+    @property
+    def moderation_role(self):
+        """Get the configured ``CURATIONS_MODERATION_ROLE`` role."""
+        return current_datastore.find_role(self.moderation_role_name)
```

Comment:
The idea of putting it on the service was that this package may be used as a base for curations and institutions may add several curation services (rdm-records, Open Education Resources, other schemas) with different configs. What do you think about that?

Reply:
Ah ok, that makes sense!
Perhaps it would then even make sense to go into the opposite direction and move the other properties from the extension to the service as well?

Reply:
Yeah the comment was for all properties that were previously on the service (lines 87 to 112).
Initializing the configs, services and resources should probably still live here as in standard invenio fashion?